molgenis-dataexplorer/src/main/java/org/molgenis/dataexplorer/settings/DataExplorerSettings.java

```diff
@@ -483,13 +480,26 @@ public void setEntityReports(String entityReportsStr)
 	@Nullable
 	public boolean isShowNavigatorLink()
 	{
-		Boolean result = getBoolean(Meta.GENERAL_NAVIGATOR_LINK);
-		return result == null ? false : result.booleanValue();
+		return isTruefy(getBoolean(Meta.GENERAL_NAVIGATOR_LINK));
 	}
 
 	@Nullable
 	public void setShowNavigatorLink(boolean showNavigatorLink)
 	{
 		set(Meta.GENERAL_NAVIGATOR_LINK, showNavigatorLink);
 	}
+
+	public boolean isUseVueDataRowEdit()
+	{
+		return isTruefy(getBoolean(Meta.GENERAL_USE_VUE_DATA_ROW_EDIT));
+	}
+
+	/**
+	 * @param input nullable boolean
+	 * @return boolean indicating if input is true of false, is input is null false is returned
```

Comment:
Eh I don't get this sentence

Suggestion:
Indicates if input is true **or** false
Returns false when input is null